### PR TITLE
feat: Implement BlobStorage for BlobBucketDBStorage

### PR DESCRIPTION
**Pull request by Xuanwo**
Fix https://github.com/toeverything/OctoBase/issues/464

### CODE REVIEW

File: libs/jwst-storage/src/storage/blobs/bucket_local_db.rs [hunk 1]
```diff
@@ -145,7 +137,22 @@ impl BlobStorage<JwstStorageError> for BlobBucketDBStorage {
         id: String,
         params: Option<HashMap<String, String>>,
     ) -> JwstResult<BlobMetadata, JwstStorageError> {
-        todo!()
+        let workspace = get_workspace(workspace);
+        let key = build_key(workspace, id);
+
+        let metadata = self.bucket_storage.op.stat(&key).await?;
+
+        Ok(BlobMetadata {
```
Comment (thorseraq) [inline libs/jwst-storage/src/storage/blobs/bucket_local_db.rs:145]: Hi, recommend only use db to query metadata, it's faster

Comment (Xuanwo) [reply libs/jwst-storage/src/storage/blobs/bucket_local_db.rs]: Fixed.

File: libs/jwst-storage/src/storage/blobs/bucket_local_db.rs [hunk 2]
```diff
@@ -39,30 +40,30 @@ impl BlobBucketDBStorage {
     pub async fn init_pool(database: &str) -> JwstStorageResult<Self> {
         todo!()
     }
+
+    /// TODO: can we merge with BlobDBStorage::metadata?
```
Comment (thorseraq) [inline libs/jwst-storage/src/storage/blobs/bucket_local_db.rs:44]: Agree with this, have tried to extract a common trait auto implementing some common db operations here https://github.com/thorseraq/OctoBase/tree/0614/init-s3, but haven't finished generics adaption 🥰

**Review by thorseraq — APPROVED** [review]

**Comment by thorseraq** [comment]
Thanks! So fast for another PR!

**Pull request merged**